### ISSUE TITLE
[FEATURE] Add recent alert to the new OSD homepage

### ISSUE THREAD

**Issue by ruanyl** [opensearch-project/alerting-dashboards-plugin#1037]
**Is your feature request related to a problem?**
Hey alert dashboards maintainers! The new homepage design requires to display a list of recent alerts on the homepage, as you can see on the screenshot:
<img width="1085" alt="image" src="https://github.com/user-attachments/assets/641379a5-83d9-45b2-bffb-420abd3befd9">

**What solution would you like?**
We released the new homepage without the alert card in 2.16 which based on top of a newly introduced content management plugin that allows multiple plugins to contribute contents to the same page.

The alert card is required for 2.17, below is a screenshot of the current version:
![image](https://github.com/user-attachments/assets/536b7e58-24b6-4c17-a695-d8d7b48a1e9f)

Basically, an alert card will be registered to homepage in the similar way as the workspace list card, you can refer to the workspace list card implementation here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/workspace/public/plugin.ts#L478

**What alternatives have you considered?**
None

**Do you have any additional context?**
Documentation of content management plugin: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7575/files

**Comment by dblock** [comment]
Catch All Triage - [1](https://github.com/dblock), [2](https://github.com/krisfreedain), [3](https://github.com/andrross)

**Comment by dblock** [comment]
@lezzago Is there a triage meeting for this repo? Please work with @krisfreedain to set one up.

**Comment by spapadop** [comment]
This feature seems very interesting!
`v2.17` is already out, but I guess this hasn't been implemented yet?
